contracts/drops/token/HolographDropERC721.sol

```diff
@@ -822,6 +824,21 @@ contract HolographDropERC721 is NonReentrant, ERC721H, IHolographDropERC721 {
     }
   }
 
+  function _payoutHolographFee(uint256 quantity) internal {
+    // Transfer protocol fee to recipient address
+    uint256 holographFee = getHolographFee(quantity);
+
+    // Payout Holograph fee
+    if (holographFee > 0) {
+      address payable holographFeeRecipient = payable(
```

Comment:
Should we add a function to see the recipient?

Reply:
I changed my mind about this. This is something only Holograph can do so it shouldn't be an admin function since that would be the creator. We can update this via upgrades